packages/sdks/src/components/render-content-variants/render-content-variants.lite.tsx

```diff
@@ -89,7 +89,7 @@ export default function RenderContentVariants(props: VariantsProviderProps) {
         customComponents={props.customComponents}
         classNameProp={`variant-${props.content?.id}`}
         parentContentId={props.content?.id}
-        isSsrAbTest={state.shouldRenderVariants}
+        isSsrAbTest={false}
```

Comment:
Is this change intentional or leftover from testing something?

Reply:
It's a mistake which is causing the build to fail! Thanks for pointing this out to me 🙏🏽